README.md

```diff
@@ -329,6 +329,23 @@ class Product < ActiveRecord::Base
 end
 ```
 
+You can specify scope name instead of where condition string.
+
+```ruby
+class Product < ActiveRecord::Base
+  belongs_to :category
+  counter_culture :category,
+      column_name: proc {|model| "#{model.product_type}_count" },
+      column_names: {
+          awesomes: :awesome_count,
+          suckys: :sucky_count
```

Comment:
I like the approach of using scopes here, but could we use the actual scope here instead of symbols? That would then look like this:
```
  awesomes: Product.awesomes,
  suckys: Product.suckys
```

Reply:
`column_names` option is defined as `{ 'where_cause' => 'counter_column_name'  }`. (I understood in this way. It is right?)
So I will extend the definition to `{ 'where_cause' or 'scope_name' => 'counter_column_name'  }`.


Sorry, I'm not sure what this mean.

```
  awesomes: Product.awesomes,
  suckys: Product.suckys
```

Do you wish extend the definition to `{ 'where_cause' => 'counter_column_name'  }` or `{ 'counter_column_name' => Model.calling_scope  }`?

If my guessing is right, I think the definition is confusing to user. 
I want to hear your opinion.


I'm not used to discussing English. I'm sorry if I'm rude.

Reply:
No worries, not rude at all. And in my example I did it the wrong way around. What I meant was, could you use the actual scope instead of the name of the scope. So it would look like this:
```
column_names: {
           Product.awesomes => :awesome_count,
           Product.suckys => :sucky_count
}
```

I'm worried about the indirection of using a symbol. What if you put a symbol and then that doesn't exist as a scope? With your implementation that would just blow up on the `send` and be confusing. It seems to me that using the scope directly would be clearer and with less potential issues.

Reply:
Wow! Its perfect idea!!

Reply:
I implemented your idea. [Check this out](https://github.com/magnusvk/counter_culture/pull/272/commits/87c87be01efda567bb4a03ad2adf1aa62fc0bee0).

Chained scope becomes possible.

```
column_names: {
           Product.awesomes.suckys => :awesome_and_sucky_count
}
```

Its so good.